include/diaspora.php

```diff
@@ -2031,7 +2092,7 @@ private function original_item($guid, $orig_author, $author) {
 			if (self::is_reshare($r[0]["body"], true))
 				$r = array();
 			elseif (self::is_reshare($r[0]["body"], false)) {
-				$r[0]["body"] = diaspora2bb(bb2diaspora($r[0]["body"]));
+				$r[0]["body"] = self::replace_people_guid(diaspora2bb(bb2diaspora($r[0]["body"])), $r[0]["author-link"]);
```

Comment:
Okay, I'm not a long-term developer here. So what are the others thinking?

cc @tobiasd @fabrixxm